src/actions/index.js

```diff
@@ -458,10 +501,108 @@ function updateOutput(
         });
     }
 
+    // Clientside hook
+    if (!isNil(clientside_function) && !isEmpty(clientside_function)) {
```

Comment:
Is there actually a case for an empty `clientside_function` (or any other value that's truthy but missing `namespace` or `function_name`) that we wouldn't want to throw an error? If not I'd just make this `if (clientside_function)` so these other cases fall into the `catch` below rather than silently continuing to the server-side section.